migrator/src/main/java/uk/nhs/digital/ps/migrator/model/hippo/Attachment.java

```diff
@@ -3,8 +3,6 @@
 import org.apache.tika.Tika;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import uk.nhs.digital.ps.migrator.PublicationSystemMigrator;
-import uk.nhs.digital.ps.migrator.config.ExecutionParameters;
```

Comment:
Nice refactoring of this area.  The PR for Migration - map keywords also uses something similar to pass execution parameters into a Taxonomy class.  Therefore perhaps merge this PR first, and I'll rebase the map keywords branch to follow this convention.

Reply:
Sounds like a plan - thanks, @ngta1!